sky/jobs/constants.py

```diff
@@ -23,8 +23,8 @@
 # jobs. See _get_launch_parallelism and _get_job_parallelism in scheduler.py.
 # We use 50 GB disk size to reduce the cost.
 CONTROLLER_RESOURCES: Dict[str, Union[str, int]] = {
-    'cpus': '4+',
-    'memory': '8x',
+    'cpus': '8+',
```

Comment:
do we need 8+ CPUs by default? can it be 4+ 4x instead? Considering the controller is much more efficient

Reply:
Let me check